src/eei.cpp

```diff
@@ -56,6 +56,12 @@ string toHex(evm_uint256be const& value) {
 
 }
 
+  void EthereumInterface::importGlobals(std::map<Name, Literal>& globals, Module& wasm) {
+    (void)globals;
+    (void)wasm;
+    HERA_DEBUG << "importGlobal " << "\n";
```

Comment:
Why 2 literals?

Reply:
Because there was `<< globals <<` in between :)

If you know an easy way to dump a map that would be nice.

Reply:
Only `for` loop.

Reply:
So should this be updated?